setParameters.m

```diff
@@ -91,13 +100,13 @@
     cfg.target.maxNbPerBlock = 2;
     cfg.target.duration = 0.05; % In secs
 
-    cfg.extraColumns = {'direction', 'speed', 'target', 'event', 'block'};
-    
+    cfg.extraColumns = {'direction', 'speed', 'target', 'event', 'block', 'keyName'};
+
 end
 
 function cfg = setKeyboards(cfg)
     cfg.keyboard.escapeKey = 'ESCAPE';
-    cfg.keyboard.responseKey = {'space'};
+    cfg.keyboard.responseKey = {'space', 't'};
```

Comment:
We will be monitoring the trigger keys on the same KbQueue as the responses so we need automate adding the value from `cfg.mri.triggeyKey` to `cfg.keyboard.responseKey`